src/isolate_create_params.rs

```diff
@@ -114,6 +114,14 @@ impl CreateParams {
     self
   }
 
+  pub fn heap_limits(mut self, min: usize, max: usize) -> Self {
```

Comment:
We need some docs here. The prose by ConfigureDefaultsFromHeapSize in v8.h would be ok if the parameter names are made to match.